components/AssetTabContent/OverviewContent.tsx

```diff
@@ -20,6 +20,14 @@ const Overview = styled.div`
   max-width: 70ch;
 `
 
+const MoreInfoContainer = styled.div<{ alone: boolean }>`
+  ${({ alone }) =>
```

Comment:
Sounds so sad #4alone 😄

Reply:
btw, I think you could just do it like this and avoid the css import
`  ${({ hasDropShadow }) => hasDropShadow && `box-shadow: var(--topBar-box-shadow);`}`